SUMMARY.md

```diff
@@ -4,4 +4,4 @@
 * [How We're Doing It](02-how-were-doing-it.md)
 * [Tips](03-tips.md)
 * [After the Session](04-after-the-session.md)
-
+* [Online Mentoring](05-online-mentoring.md)
```

Comment:
I would suggest adding this right after or before the `Tips` section, since `After the Session` would still apply for online events as well.